addons/account_audit_trail/report/audit_trail_report_views.xml

```diff
@@ -33,19 +33,24 @@
         </field>
     </record>
 
-    <record id="action_l10n_in_audit_trail_report" model="ir.actions.act_window">
-        <field name="name">Audit trail</field>
+    <record id="action_account_audit_trail_report" model="ir.actions.act_window">
+        <field name="name">Audit Trail</field>
         <field name="res_model">mail.message</field>
         <field name="view_id" ref="view_message_tree_audit_log"/>
         <field name="view_mode">tree</field>
+        <field name="help" type="html">
+            <p class="o_view_nocontent_smiling_face">
+            To see the audit log you have to enable the audit trail option from settings
+            </p>
+        </field>
         <field name="domain">[
             ('model', '=', 'account.move'),
             ('message_type', '=', 'notification'),
-            ('l10n_in_audit_log_account_move_id', '!=', False),
+            ('account_audit_log_account_move_id', '!=', False),
+            ('show_audit_log', '=', True),
         ]</field>
         <field name="search_view_id" ref="view_message_tree_audit_log_search"/>
     </record>
 
-    <menuitem id="l10n_in_audit_trail_report_menu" name="Audit trail" action="action_l10n_in_audit_trail_report" parent="l10n_in.account_reports_in_statements_menu" sequence="2"
-        groups="account.group_account_readonly"/>
+    <menuitem id="account_audit_trail_menu" name="Audit Trail" action="action_account_audit_trail_report" parent="account.account_reports_management_menu"/>
```

Comment:
We don't need the menuitem if there is a link in the config view (see feedback on the task)

Reply:
@william-andre This is accessed by the account manager.
how does the account manager go to settings and open it?

Reply:
ok keep it